doc/release-notes-1.0.0.rst

```diff
@@ -0,0 +1,322 @@
+.. _ncs_release_notes_100:
+
+|NCS| v1.0.0 Release Notes
+##########################
+
+This project is hosted by Nordic Semiconductor to demonstrate the integration of Nordic SoC support in open source projects, like MCUBoot and the Zephyr RTOS, with libraries and source code for low-power wireless applications.
+
+nRF Connect SDK v1.0.0 supports development with nRF9160 Cellular IoT devices.
+It contains references and code for Bluetooth Low Energy devices in the nRF52 Series, though development on these devices is not currently supported with the nRF Connect SDK.
+
+Highlights
+**********
+
+*
+*
+*
+*
+*
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.0.0**.
+Check the ``west.yml`` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`Getting the nRF Connect SDK code <cloning_the_repositories_win>` for more information.
+
+
+Supported boards
+****************
+
+* PCA10090 (nRF9160 DK) - only for LTE samples
```

Comment:
Do we need the note "only for LTE samples"?  It is not immediately obvious what this means.

Reply:
yes, fixed